datasets/proj-datumgrid-geotiff.yaml

```diff
@@ -0,0 +1,9 @@
+Name: PROJ datum grids in GeoTIFF formats on AWS
+Description: Horizontal and vertical adjustment datasets for coordinate transformation to be used by PROJ 7 or later
```

Comment:
Should we point to cdn.proj.org too?

Reply:
@hobu added this under the `Resources` section.